akka-projection-jdbc/src/test/scala/akka/projection/jdbc/JdbcProjectionSpec.scala

```diff
@@ -242,20 +254,26 @@ class JdbcProjectionSpec
 
   private def offsetShouldBe(
       expected: Long)(implicit offsetStore: JdbcOffsetStore[PureJdbcSession], projectionId: ProjectionId) = {
-    val offset = offsetStore.readOffset[Long](projectionId).futureValue.value
-    offset shouldBe expected
+    withClue(s"checking offset value is [$expected]") {
+      val offset = offsetStore.readOffset[Long](projectionId).futureValue.value
+      offset shouldBe expected
+    }
```

Comment:
I added back the `withClue`. 

It is useful to have it, although I agree that we were not using it in the best way. 

It's useful in cases like: 

```scala
eventually {
	projectedValueShouldBe(e1, e2, e3)
	offsetShouldBe(3)
}
```

It helps identify which one failed. Without it we only see: `The Option on which value was invoked was not defined..`. Sure we can use the stack trace, but it's easier if we see a label.

Reply:
ok, here it might give some value, but it could also be same information with:
```
offsetStore.readOffset[Long](projectionId).futureValue shouldBe Some(expected)
```

also, if using clue I think there should be an extra`:` at the end of the clue to give nice messages

Reply:
Ok, I see your point. `shouldBe Some(expected)` will provide enough context. Good point.